File: src/client/java/com/braydenoneal/BlogicClient.java
package com.braydenoneal;

import com.braydenoneal.block.entity.ModBlockEntities;
import net.fabricmc.api.ClientModInitializer;
import net.minecraft.client.gui.screen.ingame.HandledScreens;

public class BlogicClient implements ClientModInitializer {
    @Override
    public void onInitializeClient() {
        HandledScreens.register(ModBlockEntities.CONTROLLER_SCREEN_HANDLER, ControllerScreen::new);
    }
}

File: src/client/kotlin/com/braydenoneal/BlogicClient.kt
package com.braydenoneal

import com.braydenoneal.block.entity.ModBlockEntities.Companion.CONTROLLER_SCREEN_HANDLER
import net.fabricmc.api.ClientModInitializer
import net.minecraft.client.gui.screen.ingame.HandledScreens

object BlogicClient : ClientModInitializer {
    override fun onInitializeClient() {
        HandledScreens.register(CONTROLLER_SCREEN_HANDLER, ::ControllerScreen)
    }
}
